fn:data(fn:error()) instance of xs:integer
